src/webui/www/private/index.html

```diff
@@ -218,13 +218,13 @@ <h1 class="applicationTitle">qBittorrent Web User Interface <span class="version
                     <td class="statusBarSeparator"></td>
                     <td id="DHTNodes"></td>
                     <td class="statusBarSeparator"></td>
-                    <td><img id="connectionStatus" alt="Connection Status" src="images/skin/firewalled.svg" style="height: 1.5em;" /></td>
+                    <td><img id="connectionStatus" alt="Connection Status" title="Connection status" src="images/skin/firewalled.svg" style="height: 1.5em;" /></td>
```

Comment:
AFAIK values in `alt`  & `title` are for users, you'll need to wrap them in `QBT_TR`